tdrs-backend/tdpservice/data_files/migrations/0001_initial.py

```diff
@@ -13,10 +16,12 @@ class Migration(migrations.Migration):
         ("stts", "0002_auto_20200923_1809"),
         migrations.swappable_dependency(settings.AUTH_USER_MODEL),
     ]
+    replaces = [('reports','0001_initial')]
```

Comment:
Great find here! This looks like a great way to both retain the history and ensure a stable migration path on deployment.